css/beeswarm.scss

```diff
@@ -0,0 +1,316 @@
+/** Calculate luma as it is also used in OCA\Theming\Util::calculateLuma */
```

Comment:
As talked about this file is not needed and should be removed

Reply:
File removed